hpx/util/lightweight_test.hpp

```diff
@@ -59,31 +59,9 @@ struct fixture
     {
     }
 
-    void increment(counter_type c)
-    {
-        switch (c)
-        {
-            case counter_sanity:
-                ++sanity_failures_; return;
-            case counter_test:
-                ++test_failures_; return;
-            default:
-                { HPX_ASSERT(false); return; }
-        }
-    }
+    void increment(counter_type c);
```

Comment:
I thought that too, however `lightweight_test.cpp` is built into the `hpx_init` static library and thus we don't need to export the symbol.